### ISSUE TITLE
Avg moving speed is over/under Avg speed

### ISSUE THREAD

**Issue by vsevjednom-cz** [OpenTracksApp/OpenTracks#836]
**Describe the bug**
I'm a little confused.
"Avg speed" is over "Avg moving speed" when GPS tracking is active.
But: "Avg speed" is under "Avg moving speed"  when I view older activity.

Is possible to unify it? For example "Avg moving speed" over "Avg speed".

**Comment by rgmf** [comment]
I'm working on #820 and I think #820 would fix this one, right?

**Comment by vsevjednom-cz** [comment]
Yes, it looks like fix. Will be the same layout for tracking and not tracking..?

**Comment by rgmf** [comment]
> Will be the same layout for tracking and not tracking..?

They cannot be the same because Current Speed wouldn't make sense for recorded tracks, for example.

**Comment by vsevjednom-cz** [comment]
I mean only the same layout of average speed 🙃.
